src/main/cpp/chassis/HolonomicDrive.cpp

```diff
@@ -217,11 +217,14 @@ void HolonomicDrive::Run()
                 }
             }
             m_resetPathplannerTrajectory = true;
-            m_bargeHelper->IsInZone();
-            m_reefHelper->IsInZone();
+            // m_bargeHelper->IsInZone();
```

Comment:
Should we just delete this?

Reply:
Yeah we can, I thought lets keep it and make sure the drivers can adapt to the changes, but I think I am going to add another button to sperate out the algae and human player load button so we aren't messed up

Reply:
Ok, I am a little disappointed that this was working so well last week, and immediately we are re-working it.  Seems like they really didn't try it completely before saying it doesn't work.

Reply:
> Ok, I am a little disappointed that this was working so well last week, and immediately we are re-working it. Seems like they really didn't try it completely before saying it doesn't work.

Yes we can merge it.   Just an observation; not saying we don't need to make changes to make them comfortable.  Just think they need to give it some actual stick time before saying "all you need to do is ...".  Last week the robot wasn't bouncing all over the place, after changes last night, I was seeing a lot of bouncing which is bad for the elevator in addition to adding time to the cycle.  Maybe they are driving it harder, but I hate to see it going back to the top of the elevator bouncing all over the place.

Reply:
> Yeah I agree, but they are the drivers. So we have to make it so they can drive it. Do you want me to push up the deletion so we can merge this?

Yes, but the elevator wasn't bouncing around last week; at least not like I was seeing it at the end last night.   Every time it bounces around we risk the elevator and it increases the cycle time.  Hence, it would be nice if they had some actual stick time before wanting changes.   Obviously we need to make it easier for them, just not sure they've given the changes a chance before wanting changes.